verify/verify-pylint.sh

```diff
@@ -17,6 +17,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+export PYLINTHOME=$(mktemp -d)
+export PATH="${TEST_SRCDIR}/pylint:${PATH}"
+
 # TODO(fejta): all python files
```

Comment:
maybe we should turn this into an exclude list? (separate from this PR, of course)